commitizen/cz/base.py

```diff
@@ -42,8 +19,28 @@ class BaseCommitizen(metaclass=ABCMeta):
 
     def __init__(self, config: BaseConfig):
         self.config = config
-        if not self.config.settings.get("style"):
-            self.config.settings.update({"style": BaseCommitizen.default_style_config})
+        self.style = Style(self.config.settings.get("style", defaults.style))
+        self.bump_pattern: Optional[str] = self.config.settings.get(
```

Comment:
I would say the `defaults.bump_pattern` is something only for `conventional commit` that I want to move out from default and move it to conventional commit. Thus, I'd like to keep the default value of this as `None`

Reply:
This behavior should be applied to all the following default values which did not exist in `BaseCommitizen`